src/components/CharacterDetail.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import styled from "styled-components"
+import { blue, black } from "./colors"
+
+const CharacterDetail = ({ character }) => {
+  return (
+    <CharCard>
+      <h3>{character.name}</h3>
+      <img
+        src={character.image}
+        alt={`Character from Rick and Morty with the name: ${character.name}`}
+      />
+      <p>Status: {character.status}</p>
+      <p>Species: {character.species}</p>
+      <p>{character.gender}</p>
+      <p>{character.origin && character.origin.name}</p>
```

Comment:
This is correct, but you can write it shorter like this. But it is the same:

```suggestion
      <p>{character.origin?.name}</p>
```